src/main/java/org/altadoon/gt6x/features/verticalmixers/VerticalMixers.java

```diff
@@ -0,0 +1,86 @@
+package org.altadoon.gt6x.features.verticalmixers;
+
+import gregapi.data.*;
+import gregapi.oredict.OreDictMaterial;
+import gregapi.tileentity.machines.MultiTileEntityBasicMachine;
+import gregapi.tileentity.machines.MultiTileEntityBasicMachineElectric;
+import gregapi.util.CR;
+import gregapi.util.UT;
+import net.minecraft.tileentity.TileEntity;
+import org.altadoon.gt6x.common.Config;
+import org.altadoon.gt6x.common.MTEx;
+import org.altadoon.gt6x.features.GT6XFeature;
+
+import static gregapi.data.CS.*;
+import static gregapi.data.CS.T;
+
+public class VerticalMixers extends GT6XFeature {
+    public static final String FEATURE_NAME ="Vertical Mixers";
+    @Override
+    public String name() {
+        return FEATURE_NAME;
+    }
+
+    @Override
+    public void configure(Config config) {
```

Comment:
For these small empty functions I would change them into a oneliner for easier overview, especially since you don't expect to add anything there, also in the future.
so
```java
@Override public void configure(Config config) {}
```
can even remove the whitespaces between those

Reply:
fixed in [59848c2](https://github.com/BorisMulder/gt6x/pull/13/commits/59848c26b8aab28645527a3c18f15aea3da67564)